src/citrine/informatics/predictors/__init__.py

```diff
@@ -0,0 +1,12 @@
+# flake8: noqa
+from .predictor import *
+from .expression_predictor import *
+from .simple_ml_predictor import *
+from .generalized_mean_property_predictor import *
+from .graph_predictor import *
+from .ingredient_fractions_predictor import *
+from .ingredients_to_simple_mixture_predictor import *
+from .label_fractions_predictor import *
+from .molecular_structure_featurizer import *
+from .simple_mixture_predictor import SimpleMixturePredictor
```

Comment:
why import these explicitly and not with `*`? (not blocking, just curious)

Reply:
That's an oversight. Fixed